src/main/java/org/junit/runner/Description.java

```diff
@@ -222,11 +222,11 @@ public int testCount() {
         if (isTest()) {
             return 1;
         }
-        int result = 0;
+        int conut = 0;
```

Comment:
`count`?

Reply:
Oops. That's a typo when I was manually copying the system output. Sorry.

The use of count was fairly common as seen in countTestCases() in junit/framework/TestSuite.java.

Reply:
@euske I was going to fix this myself, but I can't because your pull request was made on your master branch. Please fix this and send us a new pull. This should work:

```
$ git branch -f master ce0d7c7^
$ git checkout -b consisent-variable-names ce0d7c7
$ git push --set-upstream origin consisent-variable-names 
```

Reply:
You also may want to change your `master` branch `main`. You can follow the "What Your Teammates Have to Do" instructions at https://www.git-tower.com/learn/git/faq/git-rename-master-to-main/

I suggest fixing your master branch (following my previous instructions) first